README.md

```diff
@@ -113,6 +114,13 @@ the **RUNTIME** requirements.*
 
    After following through all previous instructions, you can customize the `ROS2 Lidar Sensor` component in the Entity
    Inspector to fit all your needs.
+4. **Select Classes for Semantic Segmentation**
+
+    Using the `Semantic Classes` field, you can add tuples describing the class name, the class ID, and its color. The mapping of class IDs to names will also be published alongside point cloud data.
+    
+    **Important:** Class ID 0 is reserved for unclassified points, and class ID 1 is reserved for ground points (coming from the terrain gem).
```

Comment:
Is this maybe possible to force from UI? Or maybe it is already being done like that?

Reply:
We can leave this as an issue, @michalpelka agreed that it's ok to expect that user can read